### ISSUE TITLE
What option was chosen when setting up the dynamodbstreams?

### ISSUE THREAD

**Issue by pjiocnic** [aws-samples/aws-serverless-developer-experience-workshop-java#36]
### Summary

Prior to this page https://catalog.workshops.aws/serverless-developer-experience/en-US/unicorn-contracts/pipes, I don't see any mention of how the streams for Dynamodb was setup.  In particular, which of the available options - Key attributes only, New image, Old image or New and old images was chosen?

### Why is this needed?

It helps make a guided decision and not encounter a problem at a later stage

**Comment by sliedig** [comment]
Hello @pjiocnic,

thanks for your question. No, we did not mention it because this was something that is already implemented for you when you initialised the project using `sam init`.  We decided that it was probably better to get people moving through the workshop quicker and focus on the things we felt were important, rather than going through every decision step.

Having said that, the code we implemented for looks like this https://github.com/aws-samples/aws-serverless-developer-experience-workshop-java/blob/eb651882f621a790b2b56c3b4801efc3a540c762/unicorn_contracts/template.yaml#L240

TBH, I think we could have just chosen the `NEW` StreamViewType since what we are interest in when we are creating the `ContractStatusChanged` event from the current state of the record, but having NEW_AND_OLD_IMAGES wasn't a deal breaker. 

I'll review this with the team and we will make the change if it clarifies the intent better.
